### ISSUE TITLE
Assessment identifiers tab: access rights specification

### ISSUE THREAD

**Issue by vedina** [ideaconsult/Toxtree.js#108]
Add two more fields:

1. _“Users with write access”_, in this field, multiple Ambit users should be specified by the owner of the assessment 

2. _“Users with read access”_ in this field, multiple Ambit users should be specified by the owner of the assessment. These specified users can read the assessment and use it as template to create a own assessment. If all Ambit users are allowed to read, it should be possible to specify _“all users”_ instead of listing each user by his/her name.

Implementation
* Server side 
     [TODO](https://sourceforge.net/p/ambit/feature-requests/86/)
* Client side
    * Allow to specify users (autocomplete via user REST API ?)
    * Submit the user IDs to the [server](https://sourceforge.net/p/ambit/feature-requests/86/) 
    * All users = specific role , either fix to a single role or allow to specify roles; submit the role names to the server as above.

**Comment by gonzomir** [comment]
I added the rows in the Identifiers section and added a UI component that turns a `select` dropdown with `multiple` attribute into an autocomplete list of items. For demonstration the lists are populated with the Nobel  laureates in Chemistry from 2000 to 2014. Since the underlying `select` field can have user ids for the `option`s' `value` attribute and user names displayed, it's perfect for the situation. 

The component can also work with AJAX.

**Comment by vedina** [comment]
$('.jtox-users-select', root).tokenize(); in ui_matrix.js:111 gives

````
TypeError: undefined is not a function {stack: (...), message: "undefined is not a function"}
```

**Comment by vedina** [comment]
The UI works fine.  Will create a new ticket to enable dynamic retrieval of user names.

**Comment by vedina** [comment]
Is there a way to provide JSON source for the user list? And what format ?

**Comment by gonzomir** [comment]
Yes, the users can be populated with a JSON. The format is something like this (exact field names can be configured):
```
[
  { "value": "1", "text": "Alan J. Heeger (United States)" },
  { "value": "2", "text":  "Alan G. MacDiarmid (United States, New Zealand)" }
]
```
We can also populate the `select` elements only once before initializing the widget, in which case the JSON can be any format.

**Comment by vedina** [comment]
Let's move the discussion to #115
